browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         },
         "48": {
           "release_date": "2016-08-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/48",
+          "release_notes": "https://www.mozilla.org/firefox/android/48.0beta/releasenotes/",
           "status": "retired"
         },
         "49": {
           "release_date": "2016-09-20",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/49",
+          "release_notes": "https://www.mozilla.org/firefox/android/49.0beta/releasenotes/",
           "status": "retired"
         },
         "50": {
           "release_date": "2016-11-15",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/50",
+          "release_notes": "https://www.mozilla.org/firefox/android/50.0beta/releasenotes/",
           "status": "retired"
         },
         "51": {
           "release_date": "2017-01-24",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/51",
+          "release_notes": "https://www.mozilla.org/firefox/android/51.0beta/releasenotes/",
           "status": "retired"
         },
         "52": {
           "release_date": "2017-03-07",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/52",
+          "release_notes": "https://www.mozilla.org/firefox/android/52.0beta/releasenotes/",
           "status": "esr"
         },
         "53": {
           "release_date": "2017-04-19",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/53",
+          "release_notes": "https://www.mozilla.org/firefox/android/53.0beta/releasenotes/",
           "status": "retired"
         },
         "54": {
           "release_date": "2017-06-13",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/54",
+          "release_notes": "https://www.mozilla.org/firefox/android/54.0beta/releasenotes/",
           "status": "retired"
         },
         "55": {
           "release_date": "2017-08-08",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/55",
+          "release_notes": "https://www.mozilla.org/firefox/android/55.0beta/releasenotes/",
           "status": "retired"
         },
         "56": {
           "release_date": "2017-09-28",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/56",
+          "release_notes": "https://www.mozilla.org/firefox/android/56.0beta/releasenotes/",
           "status": "retired"
         },
         "57": {
           "release_date": "2017-11-28",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/57",
+          "release_notes": "https://www.mozilla.org/firefox/android/57.0beta/releasenotes/",
           "status": "retired"
         },
         "58": {
           "release_date": "2018-01-22",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/58",
+          "release_notes": "https://www.mozilla.org/firefox/android/58.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/58.0/releasenotes/).